From: Roland Stigge <stigge@antcom.de>
Date: Mon, 5 Sep 2016 16:23:07 +0100
Subject: makefile

Adjust Makefile to user modern input variables as set by debian/rules.
---
 Makefile.simple | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/Makefile.simple b/Makefile.simple
index dded255..0151d1f 100644
--- a/Makefile.simple
+++ b/Makefile.simple
@@ -20,16 +20,14 @@ CDEBUGFLAGS = -O
 ## Uncomment this line if you have an Xlib version that doesn't
 ## support the SHAPE extension (X11R3 or earlier).  If your libraries
 ## are really that old, the program probably won't compile anyway.
-DEFINES = -DSHAPE
+CPPFLAGS += -DSHAPE
 
-LOCAL_LIBRARIES = $(XLIBDIR) -lXt -lXmu -lXext -lX11
-CFLAGS = $(CDEBUGFLAGS) $(INCLUDES) $(DEFINES)
-SYS_LIBRARIES = -lm
+LOADLIBES += -lXt -lXmu -lXext -lX11
+LOADLIBES += -lm
 SRCS = swisswatch.c SwissWatch.c Hand.c Mark.c HandOrMark.c TableConvert.c
 OBJS = swisswatch.o SwissWatch.o Hand.o Mark.o HandOrMark.o TableConvert.o
 
 swisswatch: $(OBJS)
-	$(CC) $(CFLAGS) -o swisswatch $(OBJS) $(LOCAL_LIBRARIES) $(SYS_LIBRARIES)
 
 clean::
 	rm -f swisswatch
